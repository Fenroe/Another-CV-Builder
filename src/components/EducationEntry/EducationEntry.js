function EducationEntry() {
    return (
        <div>
            <fieldset>
                <legend>Education</legend>
                <button>Add Institute</button>
            </fieldset>
        </div>
    )
};

export default EducationEntry;
