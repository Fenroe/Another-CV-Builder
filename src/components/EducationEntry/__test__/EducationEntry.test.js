import { render, cleanup } from "@testing-library/react";
import React from "react";
import "@testing-library/jest-dom";
import EducationEntry from "../EducationEntry";

afterEach(cleanup);

it('renders', () => {
    const {asFragment} = render(<EducationEntry />);
    expect(asFragment()).toMatchSnapshot();
});
