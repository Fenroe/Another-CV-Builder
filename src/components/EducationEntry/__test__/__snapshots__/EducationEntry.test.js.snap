// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div>
    <fieldset>
      <legend>
        Education
      </legend>
      <button>
        Add Institute
      </button>
    </fieldset>
  </div>
</DocumentFragment>
`;
